tsconfig.json

```diff
@@ -20,7 +20,7 @@
     /* Module Resolution Options */
     "moduleResolution": "node",
     /* This needs to be false so our types are possible to consume without setting this */
-    "esModuleInterop": true,
+    "esModuleInterop": false,
```

Comment:
is there a chance this is false by default and we can just get rid of this setting?

Reply:
it is the default, yes. I think I'll keep it though as it'll prevent people from setting it to true in PRs. Although we'll see it during review, I think it's easier if people see it without needing that review